test/E2ETests/TestAssets/SuiteLifeCycleTestProject/SuiteLifeCycleTestClass_AssemblyInitializeAndCleanup.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SuiteLifeCycleTestProject;
+
+[TestClass]
+public class SuiteLifeCycleTestClass_AssemblyInitializeAndCleanup
+{
+    private static TestContext s_testContext;
+
+    [AssemblyInitialize]
+    public static void AssemblyInit(TestContext testContext)
+    {
+        s_testContext = testContext;
+        s_testContext.WriteLine("AssemblyInit was called");
```

Comment:
I thought the assembly init was added to the 1st test run but I didn't see it in the tests. Have I missed something?